src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDateTime;
+
+import task.Deadline;
+
+public class DeadlineTest {
+    @Test
+    public void test1() {
```

Comment:
Perhaps test method name can be more descriptive

Reply:
Fair point, thanks alot!